lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -825,10 +825,12 @@ endif
 $(DYLIB_FILENAME) : $(DYLIB_OBJECTS)
 	@echo "### Linking dynamic library $(DYLIB_NAME)"
 ifneq "$(EXCLUDE_WRAPPED_SWIFTMODULE)" ""
-	$(SWIFTC) $(patsubst -g,,$(SWIFTFLAGS)) -emit-library $(DYLIB_SWIFT_FLAGS) -o $@ \
+	$(SWIFTC) $(patsubst -g,,$(SWIFTFLAGS)) -toolchain-stdlib-rpath \
+            -emit-library $(DYLIB_SWIFT_FLAGS) -o $@ \
```

Comment:
Why do we need this option? We should run all our tests with the right DYLD_LIBRARY_PATH already

Reply:
I wasn't aware of that we did that. This seems to be better to me though, since it allows manually running and debugging the test binaries, (not just through dotest) which is something I often do.
Slightly more concerning though: I had to add this option in order to get the test in rdar://64169273 to work. That might be worth investigating.